src/test/java/com/google/api/codegen/testdata/discogapic/java/java_simplecompute.v1.json.baseline

```diff
@@ -3100,7 +3100,7 @@ public final class DeleteAddressHttpRequest implements ApiMessage {
       return newBuilder;
     }
 
-    public final Builder setAddressWithAddressName(AddressName value) {
+    public final Builder setAddress(AddressName value) {
```

Comment:
I'm confused about this renaming. Where is it coming from?

Reply:
This is because of the FeatureConfig change for Java. As a follow-up change, we need to change the input and output types of these methods to String, but I didn't want to tackle that within this PR.

Reply:
https://github.com/googleapis/toolkit/issues/1804